functions/Add-DbaComputerCertificate.ps1

```diff
@@ -54,87 +54,87 @@ Add-DbaComputerCertificate -Path C:\temp\cert.cer
+      }
+      catch {
+        Stop-Function -Message "Can't import certificate." -ErrorRecord $_ -Continue
+      }
+    }
+  }
+  process {
+    if (Test-FunctionInterrupt) { return }
 		
-		if (!$Certificate -and !$Path) {
-			Write-Message -Level Warning -Message "You must specify either Certificate or Path"
-			return
-		}
+    if (!$Certificate -and !$Path) {
+      Write-Message -Level Warning -Message "You must specify either Certificate or Path"
+      return
+    }
 		
-		if ($Certificate) {
-			try {
-				$certdata = $certificate.Export([System.Security.Cryptography.X509Certificates.X509ContentType]::PFX, $password)
-			}
-			catch {
-				Stop-Function -Message "Can't export certificate" -ErrorRecord $_ -Continue
-			}
-		}
+    if ($Certificate) {
+      try {
+        $certData = $Certificate.Export([System.Security.Cryptography.X509Certificates.X509ContentType]::PFX, $Password)
+      }
+      catch {
+        Stop-Function -Message "Can't export certificate" -ErrorRecord $_ -Continue
+      }
+    }
 		
-		foreach ($computer in $computername) {
+    foreach ($computer in $ComputerName) {
 			
-			if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
-				$Password = ((65 .. 90) + (97 .. 122) | Get-Random -Count 29 | ForEach-Object { [char]$_ }) -join "" | ConvertTo-SecureString -AsPlainText -Force
-			}
+      if (![dbavalidate]::IsLocalhost($computer) -and !$Password) {
+        $Password = ((65 .. 90) + (97 .. 122) | Get-Random -Count 29 | ForEach-Object { [char]$_ }) -join "" | ConvertTo-SecureString -AsPlainText -Force
+      }
 			
-			$scriptblock = {
-				$Store = $args[2]
-				$Folder = $args[3]
+      $scriptBlock = {
+        $Store = $args[2]
+        $Folder = $args[3]
 				
-				$cert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2
-				$cert.Import($args[0], $args[1], [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::DefaultKeySet)
-				Write-Verbose "Importing cert to $Folder\$Store"
-				$tempstore = New-Object System.Security.Cryptography.X509Certificates.X509Store($Folder, $Store)
-				$tempstore.Open('ReadWrite')
-				$tempstore.Add($cert)
-				$tempstore.Close()
+        $cert = New-Object System.Security.Cryptography.X509Certificates.X509Certificate2
+        $cert.Import($args[0], $args[1], [System.Security.Cryptography.X509Certificates.X509KeyStorageFlags]::DefaultKeySet)
+        Write-Verbose "Importing cert to $Folder\$Store"
+        $tempStore = New-Object System.Security.Cryptography.X509Certificates.X509Store($Folder, $Store)
+        $tempStore.Open('ReadWrite')
+        $tempStore.Add($cert)
+        $tempStore.Close()
 				
-				Write-Verbose "Searching Cert:\$Store\$Folder"
-				Get-ChildItem "Cert:\$store\$folder" -Recurse | Where-Object { $_.Thumbprint -eq $cert.Thumbprint }
-			}
+        Write-Verbose "Searching Cert:\$Store\$Folder"
```

Comment:
Thank you for keeping this as Write-Verbose! I don't know if it's in @wsmelton's notes, but when something is in a script block and thus will be executed remotely, it has to use native commands /cc @jpomfret @csolje